cmd/pipeline/deploy.go

```diff
@@ -250,7 +250,7 @@ func (pc *Command) waitUntilRunning(ctx context.Context, name string, action *ty
 		defer wg.Done()
 		err := pc.streamPipelineLogs(waitCtx, name, action.Name)
 		if err != nil {
-			oktetoLog.Warning("there was an error streaming pipeline logs: %v", err)
```

Comment:
@pchico83 based on your suggestion from today. This will include the log in the logs but it won't be showed to the user by default

Reply:
what does it mean `include the log in the logs` ?

Reply:
The warning was being displayed to the user as part of the command output `!  there was an error streaming preview logs: http2: server sent GOAWAY and closed the connection; LastStreamID=119, ErrCode=NO_ERROR, debug=""` running it locally, this would display that line in yellow.

`include the log in the logs` means to include it as part of the logs included in the `okteto doctor` file but not being displayed to the final user by default (if the user specify --log-leve=debug it would appear)

Reply:
I think it's that the log will be in the `log file` but won't be displayed to the user in the terminal

Reply:
@ifbyol I would also add a warning only with "pipeline logs cannot be streamed due to connectivity issues". This way, the user knows why the logs aren't streaming

Reply:
ok, makes sense, added the warning!